README.md

```diff
@@ -633,6 +633,64 @@ Which can be used like this:
 // than one type (unless you have a different domain for each type)
 var api = RestService.For<IReallyExcitingCrudApi<User, string>>("http://api.example.com/users"); 
 ``` 
+### Interface inheritance
```

Comment:
I think explaining the need in terms of different authentication needs may be confusing. Can you please use a different example?

Reply:
Sure thing, I've replaced that part with a more general description in https://github.com/reactiveui/refit/pull/633/commits/199dedf10c802a8c610d06917625a5ad8853987f.